charts/managed-identity-wallets/values.yaml

```diff
@@ -15,6 +16,9 @@ logging:
   exposed: "INFO"
 wallet:
   baseWalletBpn: "BPNL000000000000"
+  baseWalletShortDid: "MhLrwtKpZhNCzazMeofPQH"
+  baseWalletVerkey: "CHEC4PRQmP73A9UD7vQ6tnLAm9aoXLPhEtnGSMiAyVZj"
```

Comment:
Unhappy with prefilled content of lines 18 and 20. I would have expected empty values here instead of valid looking IDs and Keys. The same applys to all env specific `values-xxx.yaml` files.